src/actions/arcActions.ts

```diff
@@ -135,15 +135,36 @@ export function redeemProposal(daoAvatarAddress: string, proposalId: string, acc
   };
 }
 
-export function redeemReputationFromToken(scheme: Scheme, addressToRedeem: string) {
+export function redeemReputationFromToken(scheme: Scheme, addressToRedeem: string, redeemerAddress: Address, privateKey: string|undefined) {
```

Comment:
`redeemerAddress` is never referenced

Reply:
i'd like to know why lint didn't catch this

Reply:
good point